.changeset/great-seahorses-yell.md

```diff
@@ -0,0 +1,173 @@
+---
+'@tinacms/datalayer': minor
```

Comment:
does this need an entry for the provider package?

Reply:
Since we are still telling users to use the data layer package and this does break the types in the data layer package (upgrading will break a type check) I minor makes sense to me so users do not auto update on an `install`.